src/polyfills/datalist/datalist-en.hbs

```diff
@@ -20,7 +20,7 @@
 	<form action="#" method="get">
 		<div class="form-group">
 			<label for="city">City</label>
-			<input class="form-control" type="text" id="city" name="city" list="suggestions" />
+			<input class="form-control" type="text" autocomplete="address-level2" id="city" name="city" list="suggestions" />
```

Comment:
In order to use "address-line1", "address-line2" or "address-line3", you must be using "street-address" also.

Reply:
I just did a follow-up review. What's in use here is ``"address-level2"``, which is different than ``"address-line2"``. So the [``"street-address"`` requirement you saw on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#Values) doesn't apply.

Here's the [HTML spec's definition for ``address-level2``](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fe-autocomplete-address-level2):

> The [second administrative level](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#more-on-address-levels), in addresses with two or more administrative levels; in the countries with two administrative levels, this would typically be the city, town, village, or other locality within which the relevant street address is found

**PS:** The [HTML spec's definition for ``"address-line2"``](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fe-autocomplete-address-line2) doesn't contain any mention of ``"street-address"`` needing to be used in conjunction with it. So maybe [what you saw on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#Values) was just a strongly-worded suggestion or based on an outdated definition from the spec. IMO it would be strange to require both kinds of fields to be used together since it'd be pointless for the ``"address-line#"`` fields to exist at all if they were required to be used alongside a field that contains all of them combined (i.e. ``"street-address"``).